### PR TITLE
Koe Jia-Ju (basics-9-2) "Project 1: Scissors Paper Stone (Part 1)"

### PR DESCRIPTION
**Pull request by koejiaju**
Please fill out the survey before submitting the pull request. Thanks!

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment? 
3 days

Please fill in one error and/or error message you received while working on this assignment.

What part of the assignment did you spend the most time on?
All of it

Comfort Level (1-5): 3

Completeness Level (1-5): 4

What did you think of this deliverable?
Good

Is there anything in this code that you feel pleased about?
That I completed it!

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,4 +1,91 @@
...
+//     myOutputValue = 'You win!';
+//   }
+
+//   if(opposition == 'paper' && input == 'stone' || opposition == 'scissors' && input == 'paper' || opposition == 'stone' && input == 'scissors') {
+//     myOutputValue = 'You lose!';
+//   }
+//   return myOutputValue;
+// };
+
+// //Generate computer to pick scissor, paper or stone
```
Comment (emily-zhangg) [inline script.js:17]: good use of commenting here to tell the reader what your function does

File: script.js [hunk 2]
```diff
@@ -1,4 +1,91 @@
...
+// }
+
+//More Comfortable
+//Reverse Game
+  var myOutputValue = 'Same. Play again!';
+  var opposition = randomObject();
+  var show = message(input,opposition);
+  console.log(`opposition${opposition}`);
+  if(opposition == 'paper' && input == 'reversed scissors' || opposition == 'paper' && input == 'stone'|| opposition == 'scissors' && input == 'reversed stone' || opposition == 'scissors' && input == 'paper'|| opposition == 'stone' && input == 'reversed paper' || opposition == 'stone' && input == 'scissors'){
+    myOutputValue = `${show} <br><br> You lose! Bummer. <br><br> Now you can type "scissors" "paper" or "stone" to play another round!`;
```
Comment (emily-zhangg) [inline script.js:42]: great use of the logical connectors here!

File: script.js [hunk 3]
```diff
@@ -1,4 +1,91 @@
...
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  if(randomInteger == 0) {
+    return 'scissors';
+  }
+  if(randomInteger == 1) {
+    return 'paper';
+  }
+  if(randomInteger == 2) {
+    return 'stone';
```
Comment (emily-zhangg) [inline script.js:63]: perhaps you can consider using if and else ifs here since random integer could only be 1 number and these if statements are related

**Review by yiqiingg — COMMENTED** [review]
great work jia-ju!

**Pull request closed without merging**